internal/sbom/spdx.go

```diff
@@ -40,82 +38,189 @@ func GenerateSPDX(koVersion string, date time.Time, mod []byte, imgDigest v1.Has
 		return nil, err
 	}
 
+	mainPackageID := "SPDXRef-Package-" + strings.ReplaceAll(bi.Main.Path, "/", ".")
+
+	doc := Document{
+		Version:           "SPDX-2.2",
+		DataLicense:       "CC0-1.0",
+		ID:                "SPDXRef-DOCUMENT",
+		Name:              bi.Main.Path,
+		Namespace:         "http://spdx.org/spdxdocs/" + bi.Main.Path,
+		DocumentDescribes: []string{mainPackageID},
+		CreationInfo: CreationInfo{
+			Created:  date.Format(dateFormat),
+			Creators: []string{"Tool: ko " + koVersion},
+		},
+		Packages:      make([]Package, 0, 1+len(bi.Deps)),
+		Relationships: make([]Relationship, 0, 1+len(bi.Deps)),
+	}
+
+	doc.Relationships = append(doc.Relationships, Relationship{
+		Element: "SPDXRef-DOCUMENT",
+		Type:    "DESCRIBES",
+		Related: mainPackageID,
+	})
+
+	doc.Packages = append(doc.Packages, Package{
+		Name: bi.Main.Path,
+		ID:   mainPackageID,
+		// TODO: PackageSupplier: "Organization: " + bs.Main.Path
+		DownloadLocation: "https://" + bi.Main.Path,
```

Comment:
This isn't very likely to work. Can we make this a proxy.golang.org URL? 🤔

Reply:
We used this before 🤔 

https://github.com/google/ko/blob/a61e14dc6a45d90e90ae0bec7bf3d51e48e9b5aa/internal/sbom/spdx.go#L89

I can TAL later, my tether time is up :)